File: 백준/Bronze/2475. 검증수/검증수.cc
#include <stdio.h>

int main(void) {
	int key = 0, code;
	for (int i = 0; i < 5; i++) {
		scanf("%d", &code);
		key += code * code;
	}
	key = key % 10;
	printf("%d", key);
	return 0;
}

File: 백준/Bronze/2475. 검증수/검증수.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int code, key = 0;
        for(int i = 0; i < 5; i++) {
            code = sc.nextInt();
            key += code * code;
        }
        key = key % 10;
        System.out.println(key);
    }
}